src/rpc_client/src/error.rs

```diff
@@ -21,19 +21,33 @@ pub type Result<T> = std::result::Result<T, RpcError>;
 #[derive(Error, Debug)]
 pub enum RpcError {
     #[error("Transport error: {0}")]
-    TransportError(#[from] tonic::transport::Error),
+    TransportError(Box<tonic::transport::Error>),
```

Comment:
So `#[from]` does not work with a wrapper?

Reply:
BTW, `#[from]` also helps us to implement `source` and `provide` (like backtrace). :(

Reply:
Right... How can I manually implement these?

Reply:
I see. I can keep the `#[from]` over box and forward `impl From<InnerError>` to from box.